readme.md

```diff
@@ -1,91 +1,66 @@
+  challenge and provides a means of testing and completing it.
++ [Review](http://capstone.jonathanpli.com/evaluate)  
+  This section allows peer-reviewers to selectively comment on a user's challenge submission. Given
+  industry tools to mark and edit code, peer-reviewers will use their experience to lead users
+  onto the right path for their future challenges.
++ [Feedback](http://capstone.jonathanpli.com/feedback)
+  Finally, the user will be able to view the feedback they received from a peer reviewer in this
+  area. Comments and scoring are shown in logical order and, completing the peer-review toolkit,
+  organizes improvement comments so that the user can learn and grow.
 
-###### Advanced
-Settings for the build can be changed within the `pom.xml` file under
-the `<build>...</build>` tag.
+## Technologies
+As a code learning service, we wanted a platform that was pervasive and useful. It didn't make
+sense for a coding platform to be on mobile as one would very rarely (if ever) be developing on
+a mobile device. We opted to target the computer (desktop, laptop), and chose a website as a result.
+This not only fell into our design and developer teams' strengths, but also provided the most
+relevant technology for developers today.
 
-### Server Start
-1. Check to see if a shared screen session is already running on the server. `screen -ls`
-   + If one isn't, start one with `screen -d -m -S shared` (runs screen detached, as a new 
-   session, with the name 'shared')
-   + If one is, go to step 2.
-2. Attach to the screen. Usually `screen -x <username>/shared`, but the `screen -ls` will tell 
-   you what it's called.
-   + If the screen had to be started in step 1, do the following:
-     + `[ctrl]` + `a`, then type `:multiuser on` and enter.
-     + `[ctrl]` + `a`, then type `:acladd <user>`, replacing `<user>` with (`jonathan` or 
-       `kodlee`) to add the other user to the allowed access control list.
-3. Run the server package. `java -jar <jar name>`.
-4. Detach from the screen session. `[ctrl]` + `a` THEN `d`.
++ [Java](https://www.java.com) (language), [Spring Boot](https://projects.spring.io/spring-boot/) 
+(java web server library), [Pebble](http://www.mitchellbosecke.com/pebble/home) 
+(spring templating engine)  
+  Web stacks are fairly trivial in the sense that, any major server stack will act identical to one
+  another. For our case, we chose a stack that we had little to no experience with as, this is
+  Capstone, and we wanted to learn! Java being a language our developers were comfortable with, and
+  Spring Boot + Pebble a commonly used combination of libraries, we were confident that this backend
+  would provide the functionality required to run our service.
++ [Digital Ocean](https://www.digitalocean.com/) (server hardware)  
+  One of the most cost-effective, reliable managed hardware providers on the market today. 
++ HTML, SASS (CSS), Javascript | [jQuery](https://jquery.com/), 
```

Comment:
Done. I omitted html, css, and javascript because there isn't an entity that "owns" them.